api-contract/src/main/java/uk/gov/hmcts/payment/api/contract/CreditAccountPaymentRequest.java

```diff
@@ -38,6 +38,8 @@
     @NotEmpty
     private String description;
 
+    @NotNull
+    @Pattern(regexp = "^[0-9]{16}",message = "ccd_case_number should be 16 digit")
```

Comment:
Message should also give details about value should be numeric

Reply:
Updated validation message to "ccd_case_number should be 16 digit and numeric"